docs/assets/css/tabs.less

```diff
@@ -1,166 +1,103 @@
-}
-@media (max-width: 40.0525em) and (min-width: 40.0625em) {
-    .govuk-tabs__list {
-        margin-bottom: 30px;
-    }
-}
-
-.govuk-tabs__list-item {
-    .heading-5();
-    border-bottom: 0;
-    margin-bottom: 0;
-}
-
-.govuk-tabs__list-item::before {
-    content: "\2014 ";
-    margin-left: -25px;
-    padding-right: 5px;
-}
-
-.govuk-tabs__tab:focus {
-    outline: 1px dotted @pacific;
-    outline-offset: 3px;
-}
-
-.govuk-tabs__tab:hover {
-    background-color: @gray-5;
-}
-
-.govuk-tabs__tab:active {
-    color: @black;
-}
-
-.govuk-tabs__tab:focus {
-    color: @black;
-}
-
-.govuk-tabs__tab[aria-current="true"] {
-    color: @black;
-    text-decoration: none;
-}
-
-.govuk-tabs__panel {
-    background-color: @gray-5;
-    margin-bottom: 30px;
-}
-@media (min-width: 40.0625em) {
-    .govuk-tabs__panel {
-        border-top: 1px solid @gray;
-        border-bottom: 1px solid @gray-40;
-        margin: -3px 0 50px;
-    }
-}
-@media (min-width: 40.0625em) {
-    .js-enabled {
-
-        .govuk-tabs__list:after {
-            content: "";
-            display: block;
-            clear: both;
-        }
-
-        .govuk-tabs__list-item {
-            margin-left: 0;
-
-            &--selected {
-                a {
-                    color: @black;
-                    border-bottom: 5px solid @pacific;
-                    &:hover {
-                        border-bottom: 5px solid @pacific;
-                    }
-                }
-            }
-
-            &:first-child {
-                .govuk-tabs__tab {
-                    padding-left: 2rem;
-                }
-            }
-
-        }
-
-        .govuk-tabs__list-item::before {
-            content: none;
-        }
-
-        .govuk-tabs__title {
-            display: none;
-        }
-
-        .govuk-tabs__tab {
-            border: 0;
-            color: @gray;
-            display: block;
-            margin-right: 5px;
-            padding: 1rem 1rem calc( 1rem - 5px );
-            text-align: center;
-            &:hover {
-                border-bottom: 2px solid @gray-5;
-            }
-        }
-
-        .govuk-tabs__tab--selected:focus {
-            background-color: transparent;
-        }
-
-        .govuk-tabs__panel {
-            margin-bottom: 0;
-        }
-    }
-}
-@media (min-width: 40.0625em) and (min-width: 40.0625em) {
-    .js-enabled .govuk-tabs__panel {
-        margin-bottom: 0;
-    }
-}
-@media (min-width: 40.0625em) {
-    .js-enabled .govuk-tabs__panel--hidden {
-        display: none;
-    }
-
-    .js-enabled .govuk-tabs__panel > :last-child {
-        margin-bottom: 0;
-    }
-}
-
-.variation-code-snippet-title,
-.variation-jinja-code-snippet-title,
-.variation-implementation-title,
-.variation-specs-title {
-    .respond-to-min( 641px, {
-        .u-visually-hidden();
-    } );
-    background-color: @white;
-    border-bottom: 1px solid @gray;
+    padding: 0.3em 0.5em 0.4em;
+    border: 1px solid hsl(219, 1%, 72%);
```

Comment:
Are the `hsl` copied values from the web inspector? We have a set of [color variables](https://github.com/cfpb/design-system/blob/main/docs/assets/css/color-swatches.less#L77) (swatch examples [here](https://cfpb.github.io/design-system/foundation/color)) that we'll want to use. Looking at the prior code looks like this would be `@gray`

Same thing with the background color. Refer to the swatches and https://github.com/cfpb/design-system/blob/main/docs/assets/css/color-swatches.less#L77 for the color variable name that's appropriate.